diff --git a/source/native-plugins/Makefile b/source/native-plugins/Makefile
index 6392835..36cc027 100644
--- a/source/native-plugins/Makefile
+++ b/source/native-plugins/Makefile
@@ -20,9 +20,10 @@ ifeq ($(EXPERIMENTAL_PLUGINS),true)
 ZITA_DSP_CXX_FLAGS  = $(BUILD_CXX_FLAGS) -Wno-unused-parameter
 ZITA_DSP_CXX_FLAGS += $(shell pkg-config --cflags fftw3f)
 ZITA_UI_CXX_FLAGS   = $(BUILD_CXX_FLAGS) -Wno-ignored-qualifiers -Wno-unused-parameter -Wno-unused-result
-ZITA_UI_CXX_FLAGS  += $(shell pkg-config --cflags cairo libpng12 freetype2 x11 xft zlib)
+ZITA_UI_CXX_FLAGS  += $(shell pkg-config --cflags freetype2)
+ZITA_UI_CXX_FLAGS  += $(shell pkg-config --cflags cairo libpng freetype2 x11 xft zlib)
 ZITA_UI_LINK_FLAGS  = $(LINK_FLAGS) -lclxclient -lclthreads
-ZITA_UI_LINK_FLAGS += $(shell pkg-config --libs cairo libpng12 freetype2 x11 xft zlib)
+ZITA_UI_LINK_FLAGS += $(shell pkg-config --libs cairo libpng freetype2 x11 xft zlib)
 ZITA_UI_LINK_FLAGS += -ldl -lpthread -lrt
 endif # EXPERIMENTAL_PLUGINS
 
